CODE_OF_CONDUCT.md

```diff
@@ -1,10 +1,10 @@
 ITK Community Participation Guidelines
 ======================================
 
-Version 1.0 – Updated January, 2018
+Version 1.0 – Updated May, 2018
 
 The [Insight Software Consortium](http://www.insightsoftwareconsortium.org/)
-(ISC) as the copyright holder of the National Library of Medicine Insight
+(ISC) as the copyright holder of the Insight
 Segmentation and Registration Toolkit (ITK) has adopted the following Code of
```

Comment:
I think we decided to rename it to `The Insight Toolkit` (kicking out segmentation and registration from the name).

Reply:
Thanks for catching this. Updated the toolkit name.